android/BOINC/app/src/main/java/edu/berkeley/boinc/StatusFragment.java

```diff
@@ -136,7 +136,10 @@ private void loadLayout(Boolean forceUpdate) {
 						break;
 					case BOINCDefs.SUSPEND_REASON_USER_ACTIVE:
 						Boolean suspendDueToScreenOn = false;
-						try{ suspendDueToScreenOn = BOINCActivity.monitor.getSuspendWhenScreenOn();} catch(RemoteException e){}
+						try{ suspendDueToScreenOn = BOINCActivity.monitor.getSuspendWhenScreenOn();}
```

Comment:
Please break this one up into multiple lines to be consistent with the rest of the `try/catch` constructs. Thanks.

Reply:
Done.